deploy/scripts/src/txma/test.ts

```diff
@@ -11,7 +12,64 @@ const profiles: ProfileList = {
       preAllocatedVUs: 1,
       maxVUs: 2,
       stages: [
-        { target: 1, duration: '5s' } // Ramps up to target load
+        { target: 1, duration: '60s' } // Ramps up to target load
+      ],
+      exec: 'sendEvent'
+    },
+    sendEventSmokeTest2: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1s',
+      preAllocatedVUs: 1,
+      maxVUs: 2,
+      stages: [
+        { target: 1, duration: '60s' } // Ramps up to target load
+      ],
+      exec: 'sendEventDebug'
+    }
+  },
+  load: {
+    sendEventLoadTest1: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1s',
+      preAllocatedVUs: 1,
+      maxVUs: 350,
+      stages: [
+        { target: 10, duration: '10m' } // Ramp up to 10 iterations per second in 10 minutes
+      ],
+      exec: 'sendEvent'
+    },
+    sendEventLoadTest2: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1s',
+      preAllocatedVUs: 1,
+      maxVUs: 700,
+      stages: [
+        { target: 20, duration: '10m' } // Ramp up to 20 iterations per second in 10 minutes
+      ],
+      exec: 'sendEvent'
+    },
+    sendEventLoadTest3: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1s',
+      preAllocatedVUs: 1,
+      maxVUs: 1500,
+      stages: [
+        { target: 30, duration: '10m' } // Ramp up to 30 iterations per second in 10 minutes
+      ],
+      exec: 'sendEvent'
+    },
+    sendEventLoadTest4: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1s',
+      preAllocatedVUs: 1,
+      maxVUs: 1500,
+      stages: [
+        { target: 30, duration: '60m' } // Ramp up to 30 iterations per second in 60 minutes
```

Comment:
This scenario is the same as `sendEventLoadTest3` above, is this supposed to be a higher value?

Reply:
Never mind - just noticed to difference in duration

Reply:
Yes 10min and 60mins